sdk/appconfiguration/Azure.Data.AppConfiguration/src/Models/ConfigurationSetting.cs

```diff
@@ -39,7 +39,6 @@ public ConfigurationSetting(string key, string value, string label = null): this
         /// <param name="value">The configuration setting's value.</param>
         /// <param name="label">A label used to group this configuration setting with others.</param>
         /// <param name="etag">The ETag value for the configuration setting.</param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Curious: why make this visible when it was hidden before?

Reply:
In a nutshell, we made the wrong call in our last PR.   Please see the referenced bug for full context.